lkmpg.tex

```diff
@@ -1502,6 +1502,43 @@ \subsection{Flashing keyboard LEDs}
 Flashing keyboard LEDs can be such a solution: It is an immediate way to attract attention or to display a status condition.
 Keyboard LEDs are present on every hardware, they are always visible, they do not need any setup, and their use is rather simple and non-intrusive, compared to writing to a tty or a file.
 
+From v4.14 to v4.15, the timer API made a series of changes to improve memory safety.
+A buffer overflow in the area of a \cpp|timer_list| structure may be able to overwrite the \cpp|function| and \cpp|data| fields, providing attackers with a way to use return-object programming to call arbitrary functions within the kernel.
```

Comment:
Add `\footnote` for ROP.

Reply:
The ROP is already mentioning in the syscall section.
It absolutely cannot only put the footnote here.

Reply:
Alternatively, place appropriate `\ref`.